### PR TITLE
updating scripts, fixing and adding podspecs, running formatter

### PR DESCRIPTION
**Pull request by sbenedicadb**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: AEPSwiftUI.podspec [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    s.source       = { :git => 'https://github.com/adobe/aepsdk-ui-ios.git', :tag => s.version.to_s }
+    
+    s.platform = :ios, "15.0"
+    s.swift_version = '5.1'
+  
+    s.pod_target_xcconfig = { 'BUILD_LIBRARY_FOR_DISTRIBUTION' => 'YES' }
+  
+    s.source_files = 'Frameworks/AEPSwiftUI/Sources/**/*.swift'
+
+    s.dependency 'AEPMessaging', '>= 5.2.0', '< 6.0.0'
```
Comment (PravinPK) [inline AEPSwiftUI.podspec:21]: should this be 5.3.0?

File: Frameworks/AEPSwiftUI/Sources/ContentCards/ContentCardUIEventListening.swift [hunk 2]
```diff
@@ -31,3 +31,11 @@ public protocol ContentCardUIEventListening {
     /// - Returns: A boolean indicating whether the interaction was handled.
     func onInteract(_ card: ContentCardUI, _ interactionId: String, actionURL: URL?) -> Bool
 }
+
+/// Public extension of `ContentCardUIEventListening` protocol provides default implementations,
+/// making protocol methods optional for implementers.
+public extension ContentCardUIEventListening {
```
Comment (PravinPK) [inline Frameworks/AEPSwiftUI/Sources/ContentCards/ContentCardUIEventListening.swift:37]: 👍🏼

File: Frameworks/AEPSwiftUI/Sources/AEPSwiftUI+PublicAPI.swift [hunk 3]
```diff
@@ -14,8 +14,6 @@ import AEPMessaging
 import AEPServices
 import Foundation
 
-@objc(AEPSwiftUI)
-@objcMembers
```
Comment (PravinPK) [inline Frameworks/AEPSwiftUI/Sources/AEPSwiftUI+PublicAPI.swift:18]: thank you for removing this.
Can we remove one last one in the class `ContentCardUIError` ?

File: Makefile [hunk 4]
```diff
@@ -128,7 +128,8 @@ test-SPM-integration:
 	(sh ./Script/test-SPM.sh)
 
 test-podspec:
-	(sh ./Script/test-podspec.sh)
+	(sh ./Script/test-podspec.sh -n AEPNotificationContent -v 12.0)
+	(sh ./Script/test-podspec.sh -n AEPSwiftUI -v 15.0)
```
Comment (PravinPK) [inline Makefile:132]: Neat 💯

**Review by PravinPK — APPROVED** [review]

File: AEPSwiftUI.podspec (hunk 1 above)
Comment (sbenedicadb) [reply AEPSwiftUI.podspec:21]: once Messaging 5.3.0 is released, yes

**Pull request merged**